config/routes.rb

```diff
@@ -27,6 +27,7 @@
   resources :comments, only: :create
   resources :conferences
   resources :contributors, only: :index
+  resources :community, only: :index
```

Comment:
Since the index action has been moved, can you remove index from the call to `resources :users` in line 23 please?